src/components/OfferCard/OfferCard.js

```diff
@@ -193,7 +205,9 @@ const OfferCard = ({
           <PriceWrapperStyled>
             <SkeletonWrapper showChildren={isDataLoaded} width={80} height={30}>
               {isTrialAvailable && (
```

Comment:
as far as I know this OfferCard is used only in my account, so the isTrialAvailable will never be true. To validate and if this is true we can remove below lines